webapp-django/crashstats/crashstats/models.py

```diff
@@ -948,9 +948,6 @@ class BugzillaBugInfo(SocorroCommon):
     # This is for how long we cache the metadata of each individual bug.
     BUG_CACHE_SECONDS = 60 * 60
 
-    # How patient we are with the Bugzilla REST API
-    BUGZILLA_REST_TIMEOUT = 5  # seconds
```

Comment:
This should use the defaults, so I removed this which just complicated things and wasn't used right.